lesson7.py

```diff
@@ -0,0 +1,58 @@
+# Task1
+
+def favorite_movie():
+    movie = input("What is your favorite movie?: ")
+    print("My favorite movie is named" " " + '{' + movie + '}')
+
+
+favorite_movie()
+
+
+# Task2
+
+def make_country():
+    country_name = input("Write a country: ")
+    capital = (input("Write the capital of" " " + country_name + ":" " "))
+    your_input = {country_name: capital}
+
+    return your_input
+
+
+dictionar = make_country()
+print(dictionar)
+
+
+# Task3
+
+def make_operation():
+    return choice
+
+
+while True:
+    print("Choose the operation: ")
+    print("Addition - 1")
+    print("Subtraction - 2")
+    print("Multiplication - 3")
+    print("Exit - 4")
+
+    choice = input("Enter the number of operation: ")
+    if choice == "4":
+        print("GoodBye")
+        break
+    if choice not in "1 , 2, 3, 4":

```

Comment:
If the user inputs a space or a comma - this would break the logic (should have been `if choice not in ["1", "2", "3"]` or `if choice not in "123"`. And we don't need to account for a 4 here - we already rule it out in the previous `if`